automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,96 @@
+if ($is64bit -and $env:chocolateyForceX86 -eq $true) {
+  $installKey = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+}
+
+$userInstallKey = "HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+if (Test-Path $userInstallKey) {
+  $installKey = $userInstallKey
+}
+
+if (-not (Test-Path $installKey)) {
+  New-Item -Path $installKey | Out-Null
+}
+
+if ($pp.GitOnlyOnPath) {
+  # update registry so installer picks it up automatically
+  New-ItemProperty $installKey -Name "Inno Setup CodeFile: Path Option" -Value "Cmd" -PropertyType "String" -Force | Out-Null
+}
+
+if ($pp.WindowsTerminal) {
+  # update registry so installer picks it up automatically
+  New-ItemProperty $installKey -Name "Inno Setup CodeFile: Bash Terminal Option" -Value "ConHost" -PropertyType "String" -Force | Out-Null
+}
+
+if ($pp.GitAndUnixToolsOnPath) {
+  # update registry so installer picks it up automatically
+  New-ItemProperty $installKey -Name "Inno Setup CodeFile: Path Option" -Value "CmdTools" -PropertyType "String" -Force | Out-Null
+}
+
+if ($pp.NoAutoCrlf) {
+  # update registry so installer picks it up automatically
+  New-ItemProperty $installKey -Name "Inno Setup CodeFile: CRLF Option" -Value "CRLFCommitAsIs" -PropertyType "String" -Force | Out-Null
+}
+
+# Make our install work properly when running under SYSTEM account (Chef Cliet Service, Puppet Service, etc)
+# Add other items to this if block or use $IsRunningUnderSystemAccount to adjust existing logic that needs changing
+$IsRunningUnderSystemAccount = ([System.Security.Principal.WindowsIdentity]::GetCurrent()).IsSystem
+If ($IsRunningUnderSystemAccount)
+{
+  #strip out quicklaunch parameter as it causes a hang under SYSTEM account
+  $fileArgs = $fileArgs.Replace('icons\quicklaunch,','')
+  If ($fileArgs -inotlike "*/SUPPRESSMSGBOXES*")
+  {
+    $fileArgs = $fileArgs + ' /SUPPRESSMSGBOXES'
+  }
+}
+
+If ([bool](get-process ssh-agent -ErrorAction SilentlyContinue))
+{
+  Write-Output "Killing any git ssh-agent instances for install."
+  (Get-Process ssh-agent | Where-Object {$_.Path -ilike "*\git\usr\bin\*"}) | Stop-Process
+}
+
+$installFile = if ($is64bit -and $env:chocolateyForceX86 -eq $false) { $filePath64 } else { $filePath32 }
+
+$packageArgs = @{
+  PackageName    = 'git.install'
+  FileType       = 'exe'
+  SoftwareName   = 'Git version*'
+  File           = $installFile
+  SilentArgs     = $fileArgs
+  ValidExitCodes = @(0)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
+
+if (Test-Path $installKey) {
+  $keyNames = Get-ItemProperty -Path $installKey
+
+  if (!$pp.GitOnlyOnPath -and !$pp.GitAndUnixToolsOnPath) {
+    $installLocation = $keyNames.InstallLocation
+    if ($installLocation -ne '') {
+      $gitPath = Join-Path $installLocation 'cmd'
+      Install-ChocolateyPath $gitPath 'Machine'
+    }
+  }
+}
+
+# Lets remove the installer as there is no more need for it
+Remove-Item -Force $filePath32
```

Comment:
Add `-ea 0` on both. We don't want installer to fail if for some reason file is locked or whatever.